src/modules/launcher/Plugins/Community.PowerToys.Run.Plugin.ValueGenerator/Main.cs

```diff
@@ -150,13 +196,42 @@ private Result GetResult(IComputeRequest request)
             };
         }
 
+        private List<Result> GetSuggestionFuzzyResults(Query query, List<Result> results)
+        {
+            foreach (var generatorData in QueryHelper.GeneratorDataList)
+            {
+                var matchScore = StringMatcher.FuzzySearch(query.Search, generatorData.Keyword).Score;
+
+                if (matchScore > 0)
+                {
+                    results.Add(new Result
+                    {
+                        Title = QueryHelper.GetResultTitle(generatorData),
+                        SubTitle = QueryHelper.GetResultSubtitle(generatorData),
+                        IcoPath = GetIcoPath(),
+                        Score = matchScore,
+                        ToolTipData = new ToolTipData(QueryHelper.GetResultTitle(generatorData), QueryHelper.GetResultSubtitle(generatorData)),
+                        QueryTextDisplay = generatorData.Keyword,
```

Comment:
```suggestion
                    QueryTextDisplay = generatorData.Keyword + " ",
```

Helps the user to start typing the args immediately.